masterlist.yaml

```
@@ -23235,6 +23462,135 @@ plugins:
     inc: [ 'Paper World Map Unofficial Fix.esp' ]
     req: [ *FlatMapMarkers ]
 
+  - name: 'Requiem - ArteFakes.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64410/'
+        name: 'Requiem Patch Central'
+    after:
+      - 'Fozars_Dragonborn_-_Requiem_Patch.esp'
+      - 'Requiem - Cutting Room Floor.esp'
+      - 'Requiem - ESF Companions.esp'
+  - name: 'Requiem - Audio Overhaul Skyrim.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64410/'
+        name: 'Requiem Patch Central'
+    after:
+      - 'Fozars_Dragonborn_-_Requiem_Patch.esp'
+      - 'Requiem - Cutting Room Floor.esp'
+  - name: 'Requiem - Book Covers Skyrim.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64410/'
+        name: 'Requiem Patch Central'
+    after: [ 'Requiem - Minor Arcana - Roleplaying.esp' ]
+  - name: 'Requiem - Craftable Horse Barding.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64410/'
+        name: 'Requiem Patch Central'
+    after:
+      - 'Requiem - Immersive Horses.esp'
+      - 'Requiem - Minor Arcana - Roleplaying.esp'
+  - name: 'Requiem - Cutting Room Floor.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64410/'
+        name: 'Requiem Patch Central'
+    after: [ 'Fozars_Dragonborn_-_Requiem_Patch.esp' ]
+  - name: 'Requiem - Deadly Dragons.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64410/'
+        name: 'Requiem Patch Central'
+    after: [ 'Fozars_Dragonborn_-_Requiem_Patch.esp' ]
+  - name: 'Requiem - ESF Companions.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64410/'
+        name: 'Requiem Patch Central'
+    after: [ 'Requiem - Minor Arcana - Roleplaying.esp' ]
+  - name: 'Requiem - Flora Fixes.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64410/'
+        name: 'Requiem Patch Central'
+    after: [ 'Fozars_Dragonborn_-_Requiem_Patch.esp' ]
+  - name: 'Requiem - Heavy Armory.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64410/'
+        name: 'Requiem Patch Central'
+    after: [ 'Fozars_Dragonborn_-_Requiem_Patch.esp' ]
+  - name: 'Requiem - Immersive Horses.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64410/'
+        name: 'Requiem Patch Central'
+    after:
+      - 'Requiem - Minor Arcana - Roleplaying.esp'
+      - 'Requiem - SkyTEST.esp'
+      - 'Requiem - Wild World.esp'
+  - name: 'Requiem - Immersive Sounds Compendium.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64410/'
+        name: 'Requiem Patch Central'
+    after:
+      - 'Fozars_Dragonborn_-_Requiem_Patch.esp'
```

Sorry, I missed this the first time around:

```suggestion
    after: [ 'Fozars_Dragonborn_-_Requiem_Patch.esp' ]
```